eureka_ml_insights/user_configs/aime.py

```diff
@@ -123,14 +131,125 @@ def configure_pipeline(
             ],
             output_dir=os.path.join(self.log_dir, "eval_report"),
         )
+        
+        # Aggregate the results by a majority vote 
+        self.postevalprocess_comp = EvalReportingConfig(
+            component_type=EvalReporting,
+            data_reader_config=DataSetConfig(
+                DataReader,
+                {
+                 "path": os.path.join(self.evalreporting_comp.output_dir, "metric_results.jsonl"),
+                 "format": ".jsonl",
+                 "transform":MajorityVoteTransform(),

```

Comment:
You need a couple of column rename transforms here for the metric to be applied to your desired column:

model_output ->  original_model_output
majority_vote -> model_output

Reply:
Done